test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4246,10 +4227,12 @@ def _load_configuration(self, infile, objtype=None):
                     with open(fpath, 'w') as f:
                         f.write(print_svr)
                     file_qmgr = open(fpath)
-                    cmd = [qmgr]
                     d = self.du.run_cmd(
-                        self.hostname, cmd, stdin=file_qmgr, sudo=True,
+                        self.hostname, [qmgr], stdin=file_qmgr, sudo=True,
                         logerr=False, level=logging.DEBUG)
+                    err_msg = "Failed to load server configurations"
+                    if d['rc'] != 0:
+                        self.logger.error("%s" % err_msg)
```

Comment:
so, if this (or any of the other object load) fails, what will happen? will pbs_benchpress continue running? Should we stop and error out instead?

Reply:
Right now, it will continue and will not save the configuration for the failed command.
So, at the time of loading we will just not load any failed commands.

Reply:
shouldn't we error out if that happens?

Reply:
I agree with @agrawalravi90 we should error out if any command fails instead continue because it won't save correct conf nither it will restore correct.